roles/api-environment/files/express-api/run/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# LICENSE UPL 1.0
+#
+# Copyright (c) 2014, 2019, Oracle and/or its affiliates. All rights reserved.
+#
+# ORACLE DOCKERFILES PROJECT
+
+FROM oraclelinux:7-slim
+ARG release=18
+ARG update=5
+ARG gulp_cli_version
+# Install Oracle Instant Client
+RUN yum -y install oracle-release-el7 && yum-config-manager --enable ol7_oracle_instantclient && \
+    yum -y install oracle-instantclient${release}.${update}-basic oracle-instantclient${release}.${update}-devel oracle-instantclient${release}.${update}-sqlplus && \
+    echo /usr/lib/oracle/${release}.${update}/client64/lib > /etc/ld.so.conf.d/oracle-instantclient${release}.${update}.conf && \
+    ldconfig
```

Comment:
Just my two cents, I'd like to break every command after `&&` to a new line for better readability. For example:

```dockerfile
RUN yum -y install oracle-release-el7 && \
    yum-config-manager --enable ol7_oracle_instantclient && \
    yum -y install oracle-instantclient${release}.${update}-basic oracle-instantclient${release}.${update}-devel oracle-instantclient${release}.${update}-sqlplus && \
    echo /usr/lib/oracle/${release}.${update}/client64/lib > /etc/ld.so.conf.d/oracle-instantclient${release}.${update}.conf && \
    ldconfig
```